### PR TITLE
Modifications to support simple CGRA RTL->Layout.

### PR DESCRIPTION
**Pull request by yyan7223**
Modifications to support the CGRA-Flow generated RTL to Layout.

### CODE REVIEW

File: cgra/translate/CGRARTL_test.py [hunk 1]
```diff
@@ -37,17 +37,17 @@
...
 AddrType          = mk_bits( clog2( ctrl_mem_size ) )
 num_tiles         = width * height
-data_mem_size     = 8
+data_mem_size     = 2
 num_fu_in         = 4
 DUT               = CGRARTL
 FunctionUnit      = FlexibleFuRTL
-FuList            = [ SeqMulAdderRTL, MemUnitRTL ]#AdderRTL, MulRTL, LogicRTL, ShifterRTL, PhiRTL, CompRTL, BranchRTL, MemUnitRTL ]
-DataType          = mk_data( 32, 1 )
+FuList            = [ AdderRTL, MemUnitRTL ]#AdderRTL, MulRTL, LogicRTL, ShifterRTL, PhiRTL, CompRTL, BranchRTL, MemUnitRTL ]
```
Comment (tancheng) [inline cgra/translate/CGRARTL_test.py:49]: Plz replace the `#AdderRTL` with `# SeqMulAdderRTL, AdderRTL`.

File: cgra/translate/CGRARTL_test.py [hunk 2]
```diff
@@ -37,17 +37,17 @@
...
+DataType          = mk_data( 16, 1 )
```
Comment (tancheng) [inline cgra/translate/CGRARTL_test.py:50]: Replace this line as follows:
```
# Parameterizes channel bandwidth as around 2-byte per cycle.
payload_nbits = 16 # 32
predicate_nbits = 1
DataType = mk_data( payload_nbits, predicate_nbits )
```

**Comment by tancheng** [comment]
Hi @yyan7223, let's update this and submit?

**Comment by yyan7223** [comment]
Sorry I forget last week, and I have updated the code as you required.

**Pull request merged**